src/core_modules/capture-core/components/WidgetEventEdit/DataEntry/editEventDataEntry.actions.js

```diff
@@ -101,6 +101,10 @@ export const openEventForEditInDataEntry = ({
             type: 'DATE',
             validatorContainers: getEventDateValidatorContainers(),
         },
+        {
+            id: 'scheduledAt',
+            type: 'DATE',
```

Comment:
Does it make sense to also add here the same validator you added in the [getOpenDataEntryActions](https://github.com/dhis2/capture-app/pull/2994/files#diff-a2f9ba1fd93edf38289833e1bddf0591b4f7acde3eae1d3a90e3d68769946034R19)?